tests/Assert/Tests/AssertTest.php

```diff
@@ -1832,7 +1832,11 @@ public function testValidCallable()
     {
         $this->assertTrue(Assertion::isCallable('\is_callable'));
         $this->assertTrue(Assertion::isCallable(__NAMESPACE__.'\\Fixtures\\someCallable'));
-        $this->assertTrue(Assertion::isCallable([OneCountable::class, 'count']));
+        if (PHP_VERSION_ID >= 70400) {
+            $this->assertTrue(Assertion::isCallable([new OneCountable(), 'count']));
+        } else {
+            $this->assertTrue(Assertion::isCallable([OneCountable::class, 'count']));
+        }
```

Comment:
How about testing both a static method and an instance method?
```suggestion
        $this->assertTrue(Assertion::isCallable([new OneCountable(), 'count']));
        $this->assertTrue(Assertion::isCallable([CustomAssertion::class, 'clearCalls']));
```
(And adding `use Assert\Tests\Fixtures\CustomAssertion;` at the beginning of the file.)

This should work in all PHP versions AFAIK.

Reply:
Tested on PHP8 and it didn't work. 
`is_callable` returns `false` and type is `<ARRAY>`

Reply:
Never mind, it was a misconfiguration from a bugged version of php8 in my docker env. It works for now